packages/actions/sendMessage.js

```diff
@@ -426,10 +427,22 @@ function main(params) {
 
   var bodyData = JSON.stringify(sendMessage);
   var request = require('request');
+  var apiHost;
+  if (params.apiHost) {
+    apiHost = params.apiHost;
+  }
+  else if (params.admin_url) {
+    var adminURL = require('url').parse(`https:${params.admin_url}`);
```

Comment:
Need to check if admin url starts with http first
Today it starts with // but in future might be corrected with https://

Reply:
good idea